src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    public LocalDateTime time;
+
+    public Event(String description, LocalDateTime time) {
+        super(description);
+        this.time = time;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + time.format(DateTimeFormatter.ofPattern("MMM dd yyyy h:mma")) + ")";
```

Comment:
This line is slightly long (above the soft limit of 110 characters), perhaps wrap it across 2 lines instead?

Reply:
Will do! Thanks!